mule-deployer/src/main/java/org/mule/tools/client/standalone/controller/Controller.java

```diff
@@ -223,8 +222,8 @@ protected File getRuntimeInternalRepository() {
   }
 
   public File getLog() {
-    File logEE = newFile(osSpecificController.getMuleHome() + "/logs/mule_ee.log");
-    File logCE = newFile(osSpecificController.getMuleHome() + "/logs/mule.log");
+    File logEE = new File(osSpecificController.getMuleHome() + "/logs/mule_ee.log");
```

Comment:
The only difference here is that newFie from FileUtils was doing a .getCanonicalFile. Don't know if that will make any difference in the file created at all for this context.